tests/integration/ceph_smoke_test.go

```diff
@@ -341,3 +344,110 @@ func (s *SmokeSuite) getNonCanaryMonDeployments() ([]appsv1.Deployment, error) {
 	}
 	return nonCanaryMonDeployments, nil
 }
+
+// Smoke Test for Webhooks
+func (s *SmokeSuite) TestWebhooks() {
+	logger.Info("Create webhook Smoke Test")
+	ctx := context.TODO()
+
+	cephCRList, err := s.k8sh.RookClientset.CephV1().CephClusters(s.settings.Namespace).List(ctx, metav1.ListOptions{})
+	require.NoError(s.T(), err)
+	cephCR := cephCRList.Items[0]
+	cephCR.Spec.DataDirHostPath = "/var"
```

Comment:
move this to `t.Run` of ceph cluster as it's not required for both tests? same for other tests also. if other tests do not require variables move it to its own `t.Run`?

Reply:
I went with this [comment](https://github.com/rook/rook/pull/10376#discussion_r889017641) , since we have to add comments for all the tests so using `t.Run()`